contracts/core/ModuleCore.sol

```diff
@@ -58,7 +56,13 @@ contract ModuleCore is OwnableUpgradeable, UUPSUpgradeable, PsmCore, Initialize,
         return PairLibrary.initalize(pa, ra).toId();
     }
 
-    function initializeModuleCore(address pa, address ra, uint256 lvFee, uint256 initialDsPrice) external override onlyConfig {
+    function initialize(
+        address pa,
+        address ra,
+        uint256 lvFee,
+        uint256 initialDsPrice,
+        uint256 _psmBaseRedemptionFeePercentage
```

Comment:
i see here that `_psmBaseRedemptionFeePercentage` is not used? i think we supposed to initialize it here?